test/models/ability_test.rb

```diff
@@ -16,4 +16,27 @@ def test_admins_can_manage_all_users
 
     assert ability.can?(:manage, User.new), 'User Admin cannot manage users'
   end
+
+  def test_users_cannot_create_choices
+    user = users(:one)
+    ability = Ability.new(user)
+
+    assert ability.cannot?(:create, Choice.new), "User can create a choice"
+  end
+
+  def test_guests_can_only_read_choices
+    user = User.new
+    ability = Ability.new(user)
+
+    assert ability.can?(:read, choices(:med)), "Guest cannot read choice"
```

Comment:
Again guest user like normal user should not be able to read, create, update or delete.